tests/v2/validation/pipeline/Jenkinsfile.release.upgrade.local

```diff
@@ -172,7 +232,18 @@ node {
               stage("Build Environment") {
                 try {
                   sh "docker run -v ${validationVolume}:/root --name ${buildTestContainer} -t --env-file ${envFile} " +
-                  "${imageName} sh -c \"${workPath}pipeline/scripts/setup_environment.sh\""
+                  "${imageName} sh -c \"${workPath}pipeline/scripts/setup_environment.sh;${workPath}pipeline/scripts/build_qase_auto_testrun.sh\""
+
+                  sh "docker cp ${buildTestContainer}:${workPath}${qaseRun} ./"                  
+                  sh "./${qaseRun} --startRun=true"
+
+                  if ("${env.QASE_TEST_RUN_ID}" == "null" || "${env.QASE_TEST_RUN_ID}" == "") {
+                      echo "Getting the qase testID"
+                      env.QASE_TEST_RUN_ID = sh (  
+                        script: "./yq '.id'  ./${testRunConfig}",
```

Comment:
what if the id is empty still or is null?

Reply:
that will be checked here: https://github.com/rancher/rancher/blob/main/tests/v2/validation/pipeline/qase/testrun/main.go#L42 
If the testrun is created an ID will be created. and if ID is not present here, it throws an exception.